spec/index.bs

```diff
@@ -1027,7 +1027,7 @@ dictionary GPURenderPassDepthStencilAttachmentDescriptor {
 
     required GPULoadOp depthLoadOp;
     required GPUStoreOp depthStoreOp;
-    required float clearDepth;
+    required float clearDepth = 1.0;
```

Comment:
```suggestion
    float clearDepth = 1.0;
```

Reply:
After #283, I think this line becomes obsolete.

Reply:
If all that's left is `GPUStoreOp storeOp = "store";`, we should merge. What do you think @kainino0x @kvark ?

Reply:
sure

Reply:
Let me upload a new PR for just that change since this one is stale.